examples/waiter/src/code.rs

```diff
@@ -4,13 +4,13 @@ use crate::{
 };
 use core::ops::{Deref, DerefMut};
 use futures::future;
-use gstd::{errors::Error, exec, format, lock, msg, MessageId};
+use gstd::{ exec, format, sync, msg};
 
-static mut MUTEX: lock::Mutex<()> = lock::Mutex::new(());
-static mut MUTEX_LOCK_GUARD: Option<lock::MutexGuard<()>> = None;
-static mut RW_LOCK: lock::RwLock<()> = lock::RwLock::new(());
-static mut R_LOCK_GUARD: Option<lock::RwLockReadGuard<()>> = None;
-static mut W_LOCK_GUARD: Option<lock::RwLockWriteGuard<()>> = None;
+static mut MUTEX: sync::Mutex<()> = sync::Mutex::new(());
```

Comment:
Best practice to import directly the type from mod.
```suggestion
static mut MUTEX: Mutex<()> = Mutex::new(());
```